src/utils/router.spec.js

```diff
@@ -0,0 +1,61 @@
+import { computeHref } from './router'
+
+describe('/utils/router', () => {
+  if('works with href', async () => {
```

Comment:
We should wrap all tests related to one util into it's own describe:
`describe('computeHref()', () => { ... }) `

Reply:
That might be good.

We can nest describes.